Pkgfile

```diff
@@ -173,5 +173,10 @@ vars:
   zfs_version: 2.2.4
   zfs_sha256: 9790905f7683d41759418e1ef3432828c31116654ff040e91356ff1c21c31ec0
   zfs_sha512: 1d17e30573d594fb5c9ea77cde104616dca362fed7530296816d1b55173594f66170fcfb23ab57c27074f85b79d3eb557b4ee9a1c420e507b2434a7902d8dcc1
+
+  # renovate: datasource=git-refs versioning=git depName=https://gitlab.com/apparmor/apparmor.git
+  apparmor_version: v3.1.7
```

Comment:
there seems to be v4.0.1 available, any reason an older one was picked?

Reply:
It seems like AppArmor 4.x is not "production ready" yet. When reading the release notes around 4.x they say both "release of the AppArmor that is in development" and "many of the features in apparmor 4 with require a development kernel" https://gitlab.com/apparmor/apparmor/-/releases/v4.0.1
Most Linux distributions that include AppArmor also package AppArmor 3.x so I was thinking of waiting bumping to AppArmor 4.1 until it comes out, which they indicate  "The following AppArmor 4.1 feature release is planned to be a regular release"

Reply:
makes sense :+1: